src/System.Runtime.Caching/src/System/Runtime/Caching/_shims.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace System.Runtime.Caching
+{
+    internal sealed class PerfCounters : IDisposable
```

Comment:
Still deciding what to do with perf counters. Is it ok to keep this shim here for now?

Reply:
Yes, I think that's agood approach. It avoids introducing bugs while you don't yet have tests. We could this this way potentialy.